### ISSUE TITLE
svg export: ability to define the camera position and zoom

### ISSUE THREAD

**Issue by roipoussiere** [CadQuery/cadquery#730]
Let's start with this code:

```py
import cadquery as cq
from cadquery.cq import Workplane as WP

def fig(wp):
    cube1 = WP(wp).box(5, 5, 5)
    rod1 = WP(wp).circle(1).extrude(15)
    return cube1.union(rod1)

svg_opts = {'height': 100, 'showHidden': False, 'showAxes': False}
cq.exporters.export(fig('XY'), 'fig1.svg', opt=svg_opts)
cq.exporters.export(fig('XZ'), 'fig2.svg', opt=svg_opts)
```

![image](https://user-images.githubusercontent.com/1665542/115015857-95d50680-9eb4-11eb-9efb-c7bbb344645b.png)![image](https://user-images.githubusercontent.com/1665542/115015804-85bd2700-9eb4-11eb-9834-699a472c5671.png)

These images share the same height. But what I need is to have the same box size and position. So instead of defining svg `width` and `height` in svg export options, I would like to set the camera position and zoom.

My current hack is to add a "background plate", like this:

```py
bg = WP('ZX').transformed(offset=V(-3, 0, -5)).box(1, 17, 25)`)
```

![image](https://user-images.githubusercontent.com/1665542/115017168-61fae080-9eb6-11eb-9a1a-260d43eff33d.png)![image](https://user-images.githubusercontent.com/1665542/115017196-6de6a280-9eb6-11eb-8c69-0dcaaed0b3f9.png)

Then remove this plate in Inkscape, but it's a little bit tedious.

**Comment by jmwright** [comment]
> I would like to set the camera position and zoom.

For camera position (setting aside zoom for now), `projectDir` does not work for your use case? https://cadquery.readthedocs.io/en/latest/importexport.html#exporting-svg

**Comment by roipoussiere** [comment]
> For camera position (setting aside zoom for now), `projectDir` does not work for your use case?

No, event when using `projectDir` the boxes don't have the same size.
